packages/hash/frontend/src/blocks/page/PageTitle.tsx

```diff
@@ -59,26 +59,24 @@ export const PageTitle: VoidFunctionComponent<PageTitleProps> = ({
       return;
     }
 
-    void updateEntities([
-      {
-        accountId,
-        entityId: metadataId,
-        entityTypeId: "Page",
-        data: { title: valueToSave },
+    void updateEntity({
+      data: {
+        entityId: JSON.stringify({ accountId, entityId, entityTypeId: "Page" }),
```

Comment:
If we haven't, I think we should probably add a comment somewhere that entityId is now a stringified object of accountId, entityId and entityTypeId. I initially assumed we were just combining accountId and entityId

Reply:
Thanks for flagging - this is actually left over from an old implementation where I was using `entityTypeId` to identify pages like this. Now it's identified in the call to the hook (passing `true` to indicate it's a page we want to update), so I've removed `entityTypeId` here. It should just be `accountId` and `entityId`

Fixed in https://github.com/hashintel/hash/pull/752/commits/359c3d56034d195e626e9eaa7fe0ffdbc3079480